.github/workflows/merge.yml

```diff
@@ -593,6 +593,250 @@ jobs:
           log_file_prefix: ant_test_logs_e2e
           build: true
 
+  scratchpad:
```

Comment:
We could consider having each data type in its own workflow file, like the way there is a separate file for the file-related tests.

It's just because this `merge.yml` file is already quite large.

However, this is purely optional.